pkg/s3/bucket.go

```diff
@@ -8,6 +8,10 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 )
 
+const (
+	bucketTagKey = "velero.io/backup-location"
```

Comment:
As this is velero specific, should this be in `pkg/controller/velero/s3.go`?

Reply:
The value is needed by the S3 package. But it could be moved up to the Velero controller and passed through. I opted to keep it closer to its usage for the first pass.